engine.py

```
@@ -152,7 +152,8 @@ def _initialize_logging(self, install_root):
         else:
             rotating = logging.handlers.RotatingFileHandler(log_file, maxBytes=0, backupCount=50, delay=True)
             # Always rotate. Current user might not have the correct permission to open this file
-            rotating.doRollover() # Will open file after roll over
+            if os.path.exists(log_file):
```

Can you catch and ignore exception instead?
Stating files that should normally be there can be time consuming.